docs/src/devdocs/elements.md

```diff
@@ -8,22 +8,35 @@ by the associated reference element.
 ### Required methods to implement for all subtypes of `AbstractCell` to define a new element
 
 ```@docs
-Ferrite.reference_vertices(::Ferrite.AbstractRefShape)
+Ferrite.get_node_ids
+```
+
+Please note that if your cell holds a single tuple called `nodes` to carry the node numbers, then
+this method will work automatically. Almost all functions work automatically with the information
+provided by the reference element.
+
+Please note that if you want to implement a custom element which does not have a reference shape,
+then the relevant functions below need to be dispatched.
```

Comment:
I don't think this is possible, `AbstractCell{RT}` requires `RT<:AbstractRefShape`? 

```suggestion
```

Reply:
```
struct MyFancyDynamicCell
    nodes::Vector{Int}
end
```
?

Reply:
But that won't fit in the grid which requires
```
struct MyFancyDynamicCell <: AbstractCell{RefShape}
    nodes::Vector{Int}
end
```
?

Reply:
And I guess lot of the codebase assumes that the refshape can be determined only by the type, so not sure how an arbitrary polygon element would work. Do we already have an issue for this? Otherwise, that would be good.

Reply:
But I would leave this issue for later, as it does not have any priority for me.

Reply:
Should we then include a note that we require subtyping for our Grid? (Possibly separate PR)

Reply:
I think you already included the requirement of an AbstractCell [here](https://github.com/Ferrite-FEM/Ferrite.jl/pull/889/files#diff-b331ab3d590056374ff36772c42e8ee13544e3ce67db0ab6a3369da27e8e6659R5)?